### DISCUSSION TITLE
Google Traffic Overlay Card refresh Problem

### DISCUSSION THREAD

**Discussion by tomnick** [osmandapp/OsmAnd#18026] [Q&A]
Hi everybody,
I configured an Google Traffic Overlay Card and it works good so far. The only problem I have is, that the layer does not refresh by itself.  The workaround I have to do is to delete the tile of the layer and switch it on and off. Anybody any idea what I could have done wrong? Thanks for some help.

Google Pixel 4a with GrapheneOS
OsmAnd Version 4.4.7 Pro
Layer:
URL: http://mts0.googleapis.com/vt/lyrs=m,traffic&x={1}&y={2}&z={0}&style=15
Zoom 1-22
Expiry Time 1 Min

**Comment by DmitryAlexei** [comment]
Please see https://osmand.net/docs/user/map/raster-maps#download--update-tiles.
You can set *Expiry time* in minutes.

**Reply by Sonwon1** [reply]
I think another thing to consider is mobile data usage will climb with shorter _Expiry time_.

**Reply by tomnick** [reply]
I did set the expiry time to 1 min.

**Reply by DmitryAlexei** [reply]
@tomnick 
Probably your question is related with this issue https://github.com/osmandapp/OsmAnd/issues/17776
please add info about your observations in this topic  (#17776): describe steps to reproduce it and add screenshots

**Reply by desputinski** [reply]
Hi, is there already a feature request for OSMAND to implement traffic layer as regular addon, installable by click?
